richtext-ui/src/commonJvmAndroid/kotlin/com/halilibo/richtext/ui/util/UUID.kt

```diff
@@ -0,0 +1,7 @@
+package com.halilibo.richtext.ui.util
+
+import java.util.UUID
+
+public actual fun randomUUID(): String {
```

Comment:
Did you meant to make this public? I would expect this library, being in the business of formatting text, not to also be in the business of generating valid random UUID strings. (Any public API is API that some consumer will eventually come to rely on, that you will need to maintain.)

Reply:
Naah, should be internal.